.github/workflows/linux.yml

```diff
@@ -43,6 +43,6 @@ jobs:
       - name: Install dependencies
         run:  sudo apt-get update && sudo apt install -y $(./scripts/list-build-dependencies.sh -p apt --compiler-version 9)
       - name: Build
-        run:  ./scripts/build.sh --compiler-version 9 --lto
+        run:  ./scripts/build.sh --compiler-version 9 --release sanitize
```

Comment:
Address sanitizer is a runtime check… does it output any useful information to plain build reports? (new compiler warnings or a separate report?)

- If yes, and new output is already in build logs, then scratch my comment here
- If yes, but output is in separate log, attach that log to build artifacts, please
- If no, output is runtime only - then it shouldn't land in our CI yet - only after we'll have some rudimentary runtime tests, as in #23  - changes in build.sh are welcome, though!

Reply:
Yes, analysis only happens at runtime; so I guess that means we need some new testcases :-)  

I've added a simple startup and exit; using a virtual framebuffer to provide DOSBox and SDL a valid X $DISPLAY. 

Reports are uploaded like you've done for the static analyzer. 

I need to try it under OS X; perhaps starting dosbox via shell will pop it open on the VMs "display"? This is useful because it will exercise all the macOS and non-x86 code paths.

Given the output is so extensive, this simple test case is probably plenty enough to keep someone busy.